tests/library/Respect/Validation/Rules/HexRgbColorTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Respect\Validation\Rules;
+
+class HexRgbColorTest extends \PHPUnit_Framework_TestCase
+{
+    public function testHexRgbColorValuesONLYShouldReturnTrue()
+    {
+        $validator = new HexRgbColor();
+        $this->assertTrue($validator->assert('#0'));
+        $this->assertTrue($validator->assert('#123456'));
+        $this->assertTrue($validator->assert('#1234'));
+        $this->assertTrue($validator->assert('#1234'));
+        $this->assertTrue($validator->assert('#FFFFF'));
+        $this->assertTrue($validator->assert('123123'));
+        $this->assertTrue($validator->assert('FFFFF'));
+        $this->assertTrue($validator->assert(1));
+        $this->assertTrue($validator->assert(0));
```

Comment:
RGB triplet colors must be composed by 3 hexadecimal numbers or 3 pairs of them, so, there are only 2 allowed lengths: 3 or 6. That's because this schema is to represent the RGB channels (Red, Green, Blue), each one should not be greater them 255 (FF) and not lesser than 0 (00).